components/ww3/cime_config/buildlib_cmake

```diff
@@ -43,14 +43,23 @@ def buildlib(bldroot, installpath, case):
     # Define WW3 repository directories
     repodir  = "{}/components/ww3/src".format(srcroot)
     modeldir = "{}/WW3/model".format(repodir)
+    builddir = "{}/wav".format(exeroot)
```

Comment:
Is this the CIME EXEROOT?

Reply:
exeroot here refers to the build directory .. i.e.  in env_build.xml:    <entry id="EXEROOT" value="/lcrc/group/e3sm/ac.ethomas/E3SMv3/Build_ww3_test.G.v3.PiControl/build">

Reply:
@rljacob to clarify: this PR would then place the compiled wave code in the  "..../build/wav" directory.... if this is an inappropriate place to put the compiled WW3 code - suggestions for a better place are very welcome.

Reply:
Thats a good place!